### ISSUE TITLE
Option to indicate bundled plugin

### ISSUE THREAD

**Issue by ronichoudhury** [Kitware/tangelo#478]
As the concept of standalone Tangelo application matures, it's becoming necessary to be able to create custom plugin configuration files that only load necessary plugins.  Rather than forcing people to copy plugin materials into the application directory, there should be a way to simply instruct Tangelo to use the bundled plugins it knows about.  I believe it could look like this:

````
- name: myplugin
  enabled: true
  path: myapp/plugin/myplugin

- name: tangelo
  enabled: true
  bundled: true
````

In other words, instead of supplying the path to a plugin, you would simply say that you want a "bundled" plugin.  Then the plugin loader can simply take the standard plugin directory and construct the path itself.

This lets any application take advantage of, e.g., just the Girder and Tangelo plugins.

**Comment by ronichoudhury** [comment]
Be sure to update the documentation as well.

**Comment by ronichoudhury** [comment]
Closed via #479
